client/jetpack-cloud/sections/partner-portal/index.ts

```diff
@@ -76,7 +76,7 @@ export default function () {
 		);
 
 		page(
-			`/partner-portal/payment-method/add`,
+			`/partner-portal/payment-method/card`,
```

Comment:
Just for my own edification 😄  Why is it necessary to change this? Will the url for editing a card then be ``/partner-portal/payment-method/card/<some_id>/edit` or similar?